kafka/docker/kafka.run

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016 Confluent Inc.
```

Comment:
TBH looks a bit weird to have files like this in our repo.

Is this copied from some other repo? If yes, then maybe good idea to mention where it's copied over from.

Reply:
I took it from Harvard's Kafka stack, which I do credit: https://github.com/kontena/kontena-stacks/pull/11/files .  They obviously must have taken it from Confluent.

Reply:
I think it's ok to credit in readme, I just seemed to miss that :)